src/Inventory/Asset/InventoryAsset.php

```diff
@@ -408,7 +420,7 @@ protected function addOrMoveItem(array $input): void
         }
 
         $citem = new \Computer_Item();
-        $citem->add($input, [], false);
+        $citem->add($input, [], !$this->getItem()->isNewItem()); //log only if mainitem is not new
```

Comment:
Is the new method required? This should work:
```suggestion
        $citem->add($input, [], !$this->item->isNewItem()); //log only if mainitem is not new
```

Reply:
indeed =)